pkgs/misc/tmux-plugins/default.nix

```diff
@@ -387,8 +387,8 @@ in rec {
       sha256 = "0vavbmwhpqxk26avzbnjjswsv6cd7s0sdilwppxqjdif2zm6jg9k";
     };
     postInstall = ''
-      sed -i -e 's|fzf |${pkgs.fzf}/bin/fzf |g' $target/main.sh
-      sed -i -e 's|sed |${pkgs.gnused}/bin/sed |g' $target/main.sh
+      find $target -type f -print0 | xargs -0 sed -i -e 's|fzf |${pkgs.fzf}/bin/fzf |g'
```

Comment:
I would recommend you take a look at existing substitute functions `nixpkgs` offers https://nixos.org/manual/nixpkgs/stable/#ssec-stdenv-functions. Perhaps `substituteInPlace` could be a good fit here.

Reply:
Very cool. Thanks for that link. That seems like a much better way to go.